x-pack/plugins/ml/public/file_datavisualizer/components/import_view/importer/importer_factory.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import { CsvImporter } from './csv_importer';
+import { SstImporter } from './sst_importer';
+import { JsonImporter } from './json_importer';
+
+export function importerFactory(format, results) {
+
+  let importer;
```

Comment:
nit, as in the comment further above, this could be done without `let` by just returning the importers from within the `switch`.